hotel-admin-frontend/src/pages/HomePage.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Wrapper from '../components/Wrapper';
```

Comment:
import Wrapper from '../components'; // make a indx.js file in the ```components``` and export the components from there.